app/assets/stylesheets/components/_modal.scss

```diff
@@ -106,7 +106,8 @@
   }
 }
 
-.modal-warning {
+.modal-warning,
+.modal-jobfail {
```

Comment:
Why not stick with the more generalized warning class rather than introduce the one-use-only jobfail class?

Reply:
The class name is called dynamically by the code [depending on the error](https://github.com/18F/identity-idp/blob/master/app/views/shared/_modal_verification.slim#L3). In this case, we need a separate `jobfail` error since it has different text inside the modal.

Reply:
I see. And we don't pass the type "warning" along, right?

Reply:
Instead of basing the class on the error, what we can do is add a new `class_name` method to the `Verify::Base` view model and pass that in to the view instead of the error. That way, we can have multiple errors use the same class, which is what I think you are suggesting.

Reply:
Yes, I think so. I'm looking at it as message types or levels -- warning, error, _oh no!_ with the exact text being not particularly relevant to the presentation of the modal. Eventually I think there might be good sized stack of messages, but only a couple of types of modals.

Reply:
@tbaxter-18f I addressed your feedback here: https://github.com/18F/identity-idp/pull/1707/commits/eaa9358dcff65f11bf7afac43d58efef7987991e
Please take a look